addon/components/document-view.js

```diff
@@ -51,9 +51,21 @@ export default class DocumentViewComponent extends Component {
   @lastValue("fetchDocuments") fetchedDocuments;
   @task
   *fetchDocuments() {
+    const filter = this.args.filters || {};
+    if (
+      getOwnConfig().fetchChildCategoryDocuments?.includes(
+        this.args.filters.category,
+      )
+    ) {
+      filter.categories = this.store
+        .peekRecord("category", this.args.filters.category)
+        .children.map((category) => category.id);
+      delete filter.category;
+    }
```

Comment:
Let's rename the filter to `categories` and always filter for the selected category plus all its children.

Reply:
Done